app/views/pages/conditions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<% set_page_title(t("page_titles.routing_page")) %>
```

Comment:
Not a blocker for this PR and needs design input, but we should make sure that this page and the next one have different titles